### ISSUE TITLE
[4.0] [a11y] Inaccessible search suggestions for Keyboard users 

### ISSUE THREAD

**Issue by Puneet-Jain-18** [joomla/joomla-cms#24558]
### Steps to reproduce the issue

Goto backend
administrator/content/articles

Navigate to search bar using keyboard.
Search any popular/already searched word

try to navigate to suggestions using Keyboard.

### Expected result

User could Navigate to suggestions using keyboard

### Actual result

Unable to access suggestions

### System information (as much as possible)

Joomla! 4.0.0-alpha8-dev

**Comment by ghost** [comment]
##24152 ?

**Comment by brianteeman** [comment]
@franz-wohlkoenig no thats nothing to do with this

**Comment by brianteeman** [comment]
@Puneet-Jain-18 I must be  missing something in my understanding of your report as I dont have any problems. Maybe you can try making a screen recording to show the problem - I use Licecap for this https://www.cockos.com/licecap/

**Comment by Puneet-Jain-18** [comment]
I will do that

**Comment by Puneet-Jain-18** [comment]
![rec1](https://user-images.githubusercontent.com/34083630/55972912-0f243800-5ca2-11e9-9761-2a39139d2189.gif)

**Comment by brianteeman** [comment]
Thanks - now I understand what you mean

**Comment by Puneet-Jain-18** [comment]
Sorry I just realized I wasant using down keys

**Comment by brianteeman** [comment]
and I was just about to write that.
